packages/ide-toolbox-next/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@react-buddy/ide-toolbox-next",
+  "version": "2.3.1",
+  "description": "React IDE toolbox",
+  "main": "dist/index.js",
+  "module": "dist/index.esm.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/",
+    "LICENSE"
+  ],
+  "peerDependencies": {
+    "next": "^10.0.0",
```

Comment:
So, it will work only with next 10.x.x 
To work with the rest major versions we should use `>=` operator.
Or we should specify supported versions explicitly e.g.
    `"next": "^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0 "`